frontend/cypress.json

```diff
@@ -4,7 +4,7 @@
   "viewportHeight": 800,
   "video": false,
   "retries": {
-    "runMode": 1,
+    "runMode": 2,
```

Comment:
to avoid having to keep restarting the e2e tests less often, increasing this number increases the amount of attempts cypress will run that test if the initial run fails. in this case, it will run at most two times if the first run fails